src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900px;
+}
+
+.html {
+  font-family: "Avenir", Arial, Calibri, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+}
+
+.search-bar__input {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0,10);
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__form {
+  margin-top: 20px;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 900px;
+  line-height: normal;
+}
+
+.search-bar__input--big::before {
+  content: "";
+  background-image: url("images/Search.svg");
+  background-size: cover;
+}
+
+.search-bar__input:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar__input--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 900px;
+  line-height: normal;
+}
```

Comment:
2 point in checklist, create one base class, and one where you will override styles you need